code-challenges.test.js

```diff
@@ -56,17 +139,52 @@ var cubeAndSum2 = [0, 5, 10]
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("when sortTheExtreme is called", () => {
+    it("returns [-8, 90] when called with [3, 56, 90, -8, 0, 23, 6]", () => {
+        //Arrange 
+        const nums1 = [3, 56, 90, -8, 0, 23, 6];
+
+        //Act
+        const actualExtreme = sortTheExtreme(nums1);
+
+        //Assert
+        expect(actualExtreme).toEqual([-8, 90]);
+    })
+    it("returns [-59, 109] when called with [109, 5, 9, -59, 8, 24]", () => {
+        //Arrange 
+        const nums2 = [109, 5, 9, -59, 8, 24];
+    
+        //Act
+        const actualExtreme = sortTheExtreme(nums2);
+    
+        //Assert
+        expect(actualExtreme).toEqual([-59, 109]);
+    })
+})
+
 var nums1 = [3, 56, 90, -8, 0, 23, 6]
 // Expected output: [-8, 90]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
 
+// b) Create the function that makes the test pass.
+const sortTheExtreme = (array) => {
+    // do a sort method 
+    var sortedNum = array.sort(function(a, b) {
+    return a - b
+    })
+    // create a method that will get the first value and the last value from an array
+    getMinMax = sortedNum.filter(value=> {
```

Comment:
use of `.filter()` here makes sense but because you're only accessing two very specific values in the array you can reduce the need for iterating (via filter) by returning the specified values.  For example, 
`return [sortedNum[0], sortedNum[sortedNum.length - 1]]`